spec/controllers/concerns/idv_step_concern_spec.rb

```diff
@@ -34,6 +38,77 @@ def show
     end
   end
 
+  describe '#confirm_hybrid_handoff_needed' do
+    controller(idv_step_controller_class) do
+      before_action :confirm_hybrid_handoff_needed
+    end
+
+    let(:params) do
+      {}
```

Comment:
I think the specs would be clearer if you pass the params you want in each call to show rather than using a let. You could use a local variable if the lines get too long.

You could also change the idv_session values in the individual specs as needed, although that's more a matter of taste.